eyewear_shop/demo/stock_lot.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <record id="stock_lot_1" model="stock.lot">
+        <field name="name">0001</field>
+        <field name="product_id" ref="product_product_19"/>
+    </record>
```

Comment:
```suggestion
```

Remove unused lots...

Reply:
already been used in purchase_order_confirm.xml and all other extra lots are already been removed